contrib/babelfishpg_tsql/src/catalog.c

```diff
@@ -2181,7 +2182,7 @@ static void
 create_guest_role_for_db(char *dbname)
 {
 	const char		*guest = get_guest_role_name(dbname);
```

Comment:
Does the guest name also depends on migration_mode? If so then we should also figure this out using catalog.